mcs/swagger.yml

```diff
@@ -753,4 +798,29 @@ definitions:
       - access
     properties:
       access:
-         $ref: "#/definitions/bucketAccess"
\ No newline at end of file
+         $ref: "#/definitions/bucketAccess"
+  loginDetails:
+    type: object
+    properties:
+      loginStrategy:
+        type: string
+        enum: [form,redirect]
+      redirect:
+        type: string
+  loginRequest:
+    type: object
+    required:
+      - accessKey
+      - secretKey
+    properties:
+      accessKey:
+        type: string
+      secretKey:
+        type: string
+  loginResponse:
+    type: object
+    properties:
+      sessionId:
+        type: string
+  principal:
```

Comment:
can you add a comment of what this is for?

Reply:
done